cookbook/teaser-provider.rst

```diff
@@ -0,0 +1,120 @@
+Provider for Teaser-Selection
+=============================
+
+TeaserProvider are used to load data for Teaser-Selection. It returns an array
+of Teaser instances by an array of ids. This array can be configured by a
```

Comment:
`by an array of ids` - What exactly do you want to say with that? Not really clear to me.